color/p5js/color-contrast/index.md

```diff
@@ -0,0 +1,10 @@
+layout: p5js
+author: Edmundo Mejía Galindo
+author-url: https://github.com/edgalindo/
+title: Contrast
+tags: P5.js, "Law of Closure", "Law of Connected-Elements", "Law of Contrast", "Law of Figure-Ground", "Law of Regularity", "Law of Similarity", "Law of Simplicity", "Law of Symmetry", Interaction, Contrast
```

Comment:
Tags need to be in an Array `[]`, Laws are written without `"` the `color` tag needs to be present.   


     tags: [P5.js, Law of Closure, Law of Connected-Elements, Law of Contrast, Law of Figure-Ground, Law of Regularity, Law of Similarity, Law of Simplicity, Law of Symmetry, Interaction, Contrast, color]